shared/profile/wot-author/index.tsx

```diff
@@ -66,12 +69,15 @@ type WotModalProps = {
   submitWaiting?: boolean
 }
 
-export type Proof = {
+export type Proof = ProofLite & {
+  wotProof: RPCTypes.WotProof
```

Comment:
what's going on here? why can't wotProofs be in the object that's used in the places where we have a `ProofLite`?

Reply:
`wotProof` comes from Identify3. It is only necessary for the vouch 'creation' flow. Because it's what's used to tell the service which proofs are being vouched over. It is not necessary for 'review'. In 'review' the data comes from `wotFetchVouches`, not Identify3. It would not be a problem to make `wotFetchVouches` send up `wotProof`s but it is also not necessary.